src/component/utilModal/innerModalFrame.js/AddModal.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import "../../../css/modal.css";
+
+const datas = ["코딩팁", "채용사이트", "유용한글", "나만의 장소"];
```

Comment:
정확한 의미를 드러내는 게 좋겠습니다. 다른 부분을 담당하는 개발자가 봤을 때, 빠르게 어떤 UI에서 사용하는 값인지 파악하고 유지보수하려면요
기술과제 등 채용과정에서는 이런 네이밍이 감점요인이 되기도 합니다

```suggestion
const folderNames = ["코딩팁", "채용사이트", "유용한글", "나만의 장소"];
```

아 근데 폴더 목록도 이렇게 정적이지 않고, fetch해온 결과로 사용하셔야 될거같네요